lib/msal-node/docs/sni.md

```diff
@@ -32,26 +32,32 @@ The snippet below demonstrates how to initialize MSAL for SNI authentication:
 
 ```js
 var msal = require('@azure/msal-node');
-
-const privateKey = `-----BEGIN PRIVATE KEY-----
-/**** PRIVATE KEY CONTENT ***/
------END PRIVATE KEY-----`; 
-
-const x5c = `-----BEGIN CERTIFICATE-----
-/ *** CERTIFICATE CONTENT1 ***/
------END CERTIFICATE-----
-
------BEGIN CERTIFICATE-----
-/ *** CERTIFICATE CONTENT2 ***/
------END CERTIFICATE-----
-`; // public key certificate chain. May contain more than 1 certificate sections
+// Due to security reasons, secrets should not be hardcoded.
+// The dotenv npm package can be used to store secrets in a .env file (located in project's root directory)
+// that should be included in .gitignore.
+require('dotenv').config(); // process.env now has the values defined in a .env file
+
+// -----BEGIN PRIVATE KEY-----
+// /**** PRIVATE KEY CONTENT ***/
+// -----END PRIVATE KEY-----
+const privateKey = process.env.privateKey;
+
+// public key certificate chain. May contain more than 1 certificate sections
+// -----BEGIN CERTIFICATE-----
```

Comment:
Same comment as above. If you need folks to understand the expected cert structure, make a separate note above the example code as a specific section.